README.rst

```diff
@@ -151,6 +151,12 @@ a light weight dependency.
 ``dask.bag`` depends on ``toolz`` and ``dill``.
 
 
+Notebooks
+---------
+
+Check example notebooks using dask at https://notebooks.anaconda.org/dask/.
```

Comment:
Suggest using the term examples rather than notebooks (so that we're not restricted to just notebooks.  We might want to change the `dask-notebooks` name as well if you're ok with this.

We should probably also link to the github project here.